tests/test_reader.py

```diff
@@ -331,3 +340,84 @@ def test_read_csv(tmp_path, root):
     samples_ = read_chains(filename)
     samples_.root = samples.root
     assert_frame_equal(samples, samples_)
+
+
+def test_read_dnest():
+    np.random.seed(3)
+    ns = read_dnest4('./tests/example_data/dnest4')
+    params = ['x0', 'x1', 'logL', 'logL_birth', 'nlive']
+    assert_array_equal(ns.drop_labels().columns, params)
+    labels = [r'$x0$',
+              r'$x1$',
+              r'$\ln\mathcal{L}$',
+              r'$\ln\mathcal{L}_\mathrm{birth}$',
+              r'$n_\mathrm{live}$']
+
+    assert_array_equal(ns.get_labels(), labels)
+
+    assert isinstance(ns, DiffusiveNestedSamples)
+    assert ns.samples_at_level(9, label='x1').shape == (45, 1)
+    assert ns.plot_types() == ('visited points',)
+
+    ns.points_to_plot('visited points',
+                      label='x1',
+                      evolution=0,
+                      beta=1,
+                      base_color='C0')
```

Comment:
`test_reader.py` should be about testing the reading of files, so this and the following plotting calls are a bit out of place.